pkg/plugin/plugin.go

```diff
@@ -473,16 +473,29 @@ func (e *AutoscaleEnforcer) Filter(
 	otherResources.MarginCPU = node.otherResources.MarginCPU
 	otherResources.MarginMemory = node.otherResources.MarginMemory
 
+	// As we process all pods, we should record all the pods that either aren't
```

Comment:
Note sure what this comment is trying to say. That we should record all pods that "either aren't in `e.state.podMap` or `e.state.otherPods`"

Reply:
ah, good catch! finished that comment